cli/lib/kontena/cli/stacks/install_command.rb

```diff
@@ -45,6 +45,10 @@ def install_dependencies
         caret "Installing dependency #{pastel.cyan(dependency[:stack])} as #{pastel.cyan(target_name)}"
         cmd = ['stack', 'install', '-n', target_name, '--parent-name', stack_name]
 
+        values_from_list.each do |vals_from|
+          cmd.concat ['--values-from', vals_from + "-" + dependency['name']]
```

Comment:
Hmm... I think I would expect the `--values-from` YAML file to contain some structured `child: foo: "value"` values in this case.

It's not exactly intuitive to have `kontena stack install --values-from values.yaml` try and read the child stack values from `values.yaml-foo`?

Reply:
I think that was supposed  to read them from `values-foo.yaml` but there's something odd  going on.

Reply:
That doesn't sound sensible to me either. I wouldn't expect `kontena stack install --values-from values.yaml` to try reading for a file called `values-foo.yaml`.

Reply:
Should this also be `values_from_stack_list.each` -> `--values-from-stack`?

Looking at the code for `--values-from`, that uses the same code as `-v`, so it might even already be accepting YAML `foo.bar` keys?

Reply:
The `-v` injection to children that is done through `dependency_variables_from_options` should already do that. If you have something like `foo.bar=123` then child stack `foo` will get `-v bar=123`.